pkg/apis/tenancy/v1alpha1/types.go

```diff
@@ -221,6 +221,27 @@ type ClusterWorkspaceTypeSpec struct {
 	//
 	// +optional
 	LimitAllowedParents *ClusterWorkspaceTypeSelector `json:"limitAllowedParents,omitempty"`
+
+	// defaultAPIBindings are the APIs to bind during initialization of workspaces created from this type.
+	// The APIBinding names will be generated dynamically.
+	//
+	// +optional
+	DefaultAPIBindings []APIExportReference `json:"defaultAPIBindings,omitempty"`
+}
+
+// APIExportReference provides the fields necessary to resolve an APIExport.
+type APIExportReference struct {
+	// path is the fully-qualified path to the workspace containing the APIExport.
```

Comment:
Is there actually a concept of "unqalified workspace names" (In the analogy of domain names)? Isn't `"path to the workspace` sufficient?

Reply:
We should probably revisit all our user-facing godoc comments in follow-ups

Reply:
never mind, we use the term elsewhere